libff/algebra/curves/bls12_381/bls12_381.sage

```diff
@@ -0,0 +1,70 @@
+#!/ usr / bin / env sage - python
```

Comment:
Remove spaces in the path

Reply:
That was fixed but I must have run clang-format on the file. Will fix it again.

Reply:
Done